src/core/Plugin.php

```diff
@@ -1429,7 +1446,7 @@ public function enqueue_scripts($hook_suffix)
                 'publishpress-authors'
             ),
             'mapped_author_nonce'           => wp_create_nonce("mapped_author_nonce"),
-            'home_url'                      => home_url(),
+            'term_author_slug'              => esc_url_raw($term_author_slug),
```

Comment:
Same thing as above... the param do not stores the slug, but the link. It should have a more clear name.